test/e2e_node/kubelet_test.go

```diff
@@ -116,6 +117,64 @@ var _ = framework.KubeDescribe("Kubelet", func() {
 			Expect(err).To(BeNil(), fmt.Sprintf("Error deleting Pod %v", err))
 		})
 	})
+	Context("when scheduling a busybox Pod with hostAliases", func() {
+		podName := "busybox-host-aliases" + string(uuid.NewUUID())
+		expectedHostsFileContent := `# Kubernetes-managed hosts file.
+127.0.0.1	localhost
```

Comment:
Do we really need to check the entire file? It should be sufficient if `/etc/hosts` contains the two lines we care about.